other/utils/twitter.js

```diff
@@ -0,0 +1,36 @@
+/* eslint camelcase:0 */
+
+import Twitter from 'twitter'
+
+const TW = new Twitter({
+  consumer_key: process.env.TW_CONSUMER_KEY,
+  consumer_secret: process.env.TW_CONSUMER_SECRET,
+  access_token_key: process.env.TW_ACCESS_TOKEN_KEY,
+  access_token_secret: process.env.TW_ACCESS_TOKEN_SECRET,
+})
+
+function getProfileImageURL(params) {
+  return new Promise((resolve, reject) => {
+    TW.get('users/show.json', params, (error, data) => {
+      if (!error) {
+        resolve(data.profile_image_url.replace('_normal', ''))
+      } else {
+        reject(error)
+      }
+    })
+  })
+}
+
+function sendTweet(message) {
```

Comment:
Hi @kentcdodds. I did some clean up in the PR and added this method to send tweets automatically. I tested it and it works, I think you can save another minute here. I haven't integrated it with the  `get-up-tweet` yet because I want to get your opinion first. 

I thought of this flow. If the tweet is longer than the limit we can use the function you have to copy the tweet and edit it. Otherwise, we can use this function to tweet it automatically. 

Let me know your thoghuts. If you think it can help you or if I should remove it and stick only with the image task.